### DISCUSSION TITLE
v3.0.0-beta.5

### DISCUSSION THREAD

**Discussion by sserrata** [PaloAltoNetworks/docusaurus-openapi-docs#703] [Announcements]
## What's Changed
* Add Callbacks Support to V3 by @lucasassisrosa in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/700
* Add support to x-codeSamples in v3 by @lucasassisrosa in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/701
* Prepare release v3.0.0-beta.5 by @sserrata in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/702

**Full Changelog**: https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/compare/v3.0.0-beta.4...v3.0.0-beta.5

<hr /><em>This discussion was created from the release <a href='https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/releases/tag/v3.0.0-beta.5'>v3.0.0-beta.5</a>.</em>

**Comment by rico0610** [comment]
Will this be soon become compatible with docusaurus v3?
